packages/loader/container-loader/src/containerContext.ts

```diff
@@ -103,6 +104,13 @@ export class ContainerContext implements IContainerContext {
         return this.container.clientId;
     }
 
+    /** @deprecated Added back to unblock 0.56 integration */
```

Comment:
let's make sure there's an issue tracking resolving the issues that forced us to add this back and include that in the comment here so this doesn't get lost